docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,512 @@
+are available in the ASF archive.
+
+### Submitting On-Demand OPERA RTC-S1 Jobs
+
+!!! tip "OPERA RTC-S1 On Demand not yet available in Vertex"
+
+    OPERA RTC-S1 On Demand jobs can be submitted using the [HyP3 Python SDK](../using/sdk.md){target=_blank} 
+    or [HyP3 API](../using/api.md){target=_blank}. Support in Vertex is coming soon!
+
+To order an On-Demand OPERA RTC-S1 product, a user must pass a **Sentinel-1 Burst SLC granule for the co-polarized 
+return** (either VV or HH) as input to either the 
+[HyP3 Python SDK](../using/sdk.md){target=_blank} 
+or 
+[HyP3 API](../using/api.md){target=_blank}. If a cross-pol burst is passed as input, an error will be returned.
+
+The output product will include RTC-S1 rasters for all available polarizations (generally VV and VH or HH and HV), 
+so requiring users to pass a co-polarized burst as input ensures that there will not be accidental duplication of 
+processing effort if both the co-pol and cross-pol inputs for the same Sentinel-1 burst are submitted for processing. 
+
+To ensure successful processing, review the 
+[date range](#date-range-for-on-demand-opera-rtc-s1-products "Jump to Date Range section in this document" ) and 
+[spatial coverage](#spatial-coverage-for-on-demand-opera-rtc-s1-products "Jump to Spatial Extent section in this document" ) 
+limitations below. 
+
+#### Date Range for On-Demand OPERA RTC-S1 Products
+
+The OPERA RTC-S1 code requires that input Sentinel-1 SLCs were processed using ESA's Sentinel-1 Instrument Processing 
+Facility (IPF) version 2.70, implemented April 13, 2016, or newer. As such, **we do not support On-Demand processing 
+for Sentinel-1 acquisitions prior to April 14, 2016**. Jobs submitted for earlier bursts will return an error. 
+
+All supported Sentinel-1 acquisitions since January 1, 2022, have been processed to OPERA RTC-S1 products by the OPERA 
+project, and are already available for download from ASF's archive. Any on-demand jobs submitted for acquisitions on 
+or after that date will also return an error. 
+
+#### Spatial Coverage for On-Demand OPERA RTC-S1 Products
+
+On-Demand OPERA RTC-S1 Products can only be ordered for bursts that have associated 
+[OPERA RTC-S1 Static Layer Files](#l2-radiometric-terrain-corrected-static-layer-rtc-static-files "Jump to the Static Layer Files section of this document" ) 
+available in ASF's archive. The OPERA project supports processing over all global landmasses except for Antarctica. 
+Only bursts from 
+[Sentinel-1 IW SLC](https://sentiwiki.copernicus.eu/web/s1-products "Sentinel-1 Products" ){target=_blank} 
+products are supported as input. 
+
+Any jobs submitted where the Sentinel-1 burst is from an EW SLC or does not have associated static layers available 
+will return an error. 
+
+!!! warning "OPERA RTC-S1 On Demand coverage not available in some arctic regions"
```

Comment:
```suggestion
!!! warning "OPERA RTC-S1 On-Demand coverage not available in some arctic regions"
```

Reply:
Hmm, this is an interesting one. In my head while writing, I hadn't thought of specifically the "On Demand" part to be modifying 'coverage', but rather the idea of the "OPERA RTC-S1 On Demand" service not having coverage there. But this other reading would also be appropriate, as OPERA RTC-S1 coverage itself *is* available in some areas, but the On Demand service is not. I'll go with it.